main.js

```diff
@@ -36,6 +36,48 @@ var formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+function ODate(rootEl) {
```

Comment:
I would call it `Date`, that's what we've done in the other modules. We only add the o when requiring it `const oDate = require('o-date');`